.github/workflows/actions/deploy-aks-environment/action.yml

```
@@ -0,0 +1,34 @@
+name: Deploy AKS environment
+
+inputs:
+  environment_name:
+    description: 'The name of the environment'
+    required: true
+  api_docker_image:
+    description: 'The API Docker image to deploy to the environment'
+    required: true
+  azure_credentials:
+    description: 'JSON object containing a service principal that can deploy to the environment'
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: 1.5.0
+        terraform_wrapper: false
+
+    - uses: DFE-Digital/github-actions/set-arm-environment-variables@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
```

```suggestion
        azure-credentials: ${{ inputs.azure_credentials }}
```

The input variable is named `azure_credentials`. (Although the GitHub convention seems to be to use dashes).